src/index.ts

```diff
@@ -0,0 +1,78 @@
+const inputElement = document.getElementById("commentInput");
+const formElement = document.getElementById("inputForm");
+const ulElement = document.getElementById("commentsList");
+
+//submit form
+if (formElement !== null) {
+    formElement?.addEventListener("submit", (event) => {
+        event.preventDefault();
+        if (inputElement !== null) {
+            const newMsg = (inputElement as HTMLInputElement).value;
+            if (newMsg === "") {
+                return;
+            }
+            // li father
+            const newLiElement = document.createElement("li");
+            // profile pic
+            const newImgElement = document.createElement("img");
+            newImgElement.className = "profilePicture";
+            newImgElement.src = "https://cdn.pixabay.com/photo/2021/05/20/11/58/beauty-6268460_1280.jpg";
+            newImgElement.alt = "profile Picture";
+            // comment 
+            const newCommentElement = document.createElement("div");
+            newCommentElement.className = "comonComment";
+            //comment Header
+            const newCommentHeaderElement = document.createElement("div");
+            const newCommentHeaderAndContentElement = document.createElement("div");
+            const newCommentBodyElement = document.createElement("span");
+            newCommentHeaderElement.className = "commentHeader";
+            newCommentBodyElement.innerText = ` ${newMsg}`;
+            const newCommentHeaderBoldElement = document.createElement("b");
+            newCommentHeaderBoldElement.innerText = "tonytoscani";
```

Comment:
This should be in a method that accepts a user object (imgSrc, name etc..) and message to post